bfxapi/rest/bfx_rest.py

```diff
@@ -204,6 +204,83 @@ async def get_derivative_statuses(self, symbols):
         status = await self.fetch(endpoint)
         return status
 
+    async def get_public_pulse_hist(self, end, limit=15):
```

Comment:
end is afaik optional?

Reply:
yes, confirmed:         

```py
endpoint = f"pulse/hist?limit={limit}"
````

also works

Reply:
done!

limit = 100 as default would be better, but there is an issue with large response ( failed with status 500 - ["error",10001,"ERR_REQUEST_GENERIC: ESOCKETTIMEDOUT"] )